lib/tasks/active_stash.rake

```diff
@@ -27,7 +27,23 @@ namespace :active_stash do
     info("")
     info("")
     info("")
-    info("Opening your browser to complete signup.......")
+    info("")
+    info("")
+    info("You are being redirected to https://console.cipherstash.com/?signup=active-stash to complete your signup..")
+    info("")
+    info("")
+    info("NEXT STEPS:")
+    info("")
+    info("1. Signup with your github account or email.")
+    info("")
+    info("")
+    info("2. Grab your workspace id from the signup confirmation page")
+    info("")
+    info("")
+    info("3. Login using this rake command with your workspace id: rake active_stash:login[<your workspace id>]")
```

Comment:
```suggestion
    info("3. Log in using this Rake command with your workspace ID: rake active_stash:login[<your workspace ID>]")
```
A couple more nits: 
* Same rule as above applied to "log in" vs "login"
* Capitalisation for ID